atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -96,7 +98,11 @@ public static DbKvs create(DbKeyValueServiceConfig config) {
         ReentrantManagedConnectionSupplier connSupplier = new ReentrantManagedConnectionSupplier(connManager);
         SqlConnectionSupplier sqlConnSupplier = getSimpleTimedSqlConnectionSupplier(connSupplier);
 
-        return new DbKvs(config, config.tableFactorySupplier().get(), sqlConnSupplier);
+        DbKvs dbKvs = new DbKvs(config, config.tableFactorySupplier().get(), sqlConnSupplier);
+
+        dbKvs.init();
+
+        return dbKvs;
```

Comment:
My hope is to in time deprecate the internal configuration code path entirely to avoid this issue.